sdk/storage/azblob/container/client.go

```diff
@@ -160,6 +160,18 @@ func (c *Client) Delete(ctx context.Context, options *DeleteOptions) (DeleteResp
 	return resp, err
 }
 
+// Restore operation restore the contents and properties of a soft deleted container to a specified container.
+// For more information, see https://docs.microsoft.com/en-us/rest/api/storageservices/restore-container.
+func (c *Client) Restore(ctx context.Context, deletedContainerName string, deletedContainerVersion string) (RestoreResponse, error) {
```

Comment:
Our guidelines state that APIs should have an options type as the last argument.  This is to provide extensibility without a breaking change.

Also, are both of these args mandatory?  I see they're not in the generated code.  I know that sometimes the generated code is incorrect due to OpenAPI spec bugs.  Is that the case here?

Reply:
Converted the args to options